Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.10.8-buster
+
+# Create user with a home directory
+ARG NB_USER="draco2"
+ARG NB_UID="1000"
+ENV USER ${NB_USER}
+ENV HOME /home/${NB_USER}
+
+RUN adduser --disabled-password \
+    --gecos "Default user" \
+    --uid ${NB_UID} \
+    ${NB_USER}
+WORKDIR ${HOME}/app
+
+# Install dependencies using Poetry
+RUN python -m pip install --upgrade pip &&  \
+    pip install poetry
+COPY pyproject.toml poetry.lock ./
+# Purposely not using --no-dev here to install dev dependencies needed for Jupyter Notebooks
+RUN poetry config virtualenvs.create false && \
+    poetry install && \
+    pip install git+https://github.com/cmudig/draco2.git#egg=draco  # Install Draco2 from git, needed for notebooks
```

Comment:
Is that better than installing the sources from this repo directly?

Reply:
The only reason I chose this approach instead of installing locally from source using `python -m pip install -e .` is that I wanted to have all the dependency installation commands defined in one layer, before copying the source code with `COPY . .`. But as I think about it, we would spare the `git clone` networking with the local installation and we would ensure that package contents we access with `import draco` are actually sourced from the latest version of the code, instead of a maybe-cached dependency layer.

Reply:
I addressed this in a6e9ff4, thanks for the catch!